docs/components/store.md

```diff
@@ -126,24 +126,6 @@ Flags:
 
 ```
 
-## Time based partitioning
```

Comment:
Consider adding a link here to your new document

Reply:
OK.